app/experimenter/experiments/serializers.py

```diff
@@ -324,7 +324,7 @@ class ExperimentRecipeSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Experiment
-        fields = ("action_name", "name", "filter_object", "comment", "arguments")
+        fields = ("action_name", "name", "filter_object", "comment", "arguments", "slug")
```

Comment:
In the serializer we want it to be labeled as `experimenter_slug` but the field on the model is `slug` so you gotta do it like this:

https://github.com/mozilla/experimenter/blob/master/app/experimenter/experiments/serializers.py#L291